### PR TITLE
removed nowrap shortcode, dynamically apply timestamp styling

### PR DESCRIPTION
**Pull request by sanderson**
I initially added the `{{% nowrap %}}` shortcode to use around timestamps in tables to prevent them from wrapping to the next line. At some point, this shortcode broke how tables rendered. This PR:

- Removes the `nowrap` shortcode
- Uses JS to dynamically apply styles to tables with `_time`, `_start`, and `_stop` columns

---

- [x] Rebased/mergeable

### CODE REVIEW

File: assets/js/content-interactions.js [hunk 1]
```diff
@@ -153,3 +153,30 @@ $('.tooltip').each( function(){
   $toolTipElement = $('<div/>').addClass('tooltip-container').append($toolTipText);
```
Comment (jstirnaman) [inline assets/js/content-interactions.js:153]: It would be nice to do this without jQuery.

File: assets/js/content-interactions.js [hunk 2]
```diff
@@ -153,3 +153,30 @@ $('.tooltip').each( function(){
...
   $(this).prepend($toolTipElement);
 });
+
+/////////////////// Style time columns in tables to not wrap ///////////////////
+
+$('.article--content table').each(function() {
+  var table = $(this)
```
Comment (jstirnaman) [inline assets/js/content-interactions.js:160]:
```suggestion
  var table = $(this);
```

File: assets/js/content-interactions.js [hunk 3]
```diff
@@ -153,3 +153,30 @@ $('.tooltip').each( function(){
...
+  var timeColumns = ['_time', '_start', '_stop']
```
Comment (jstirnaman) [inline assets/js/content-interactions.js:161]:
```suggestion
  var timeColumns = ['_time', '_start', '_stop'];
```

File: assets/js/content-interactions.js (hunk 2 above)
Comment (jstirnaman) [reply assets/js/content-interactions.js:160]: Either way, I'm agnostic.

File: assets/js/content-interactions.js [hunk 4]
```diff
@@ -153,3 +153,30 @@ $('.tooltip').each( function(){
...
+
+  // Return indexes of time columns
+  header.forEach(function(value, i) {
+    if ( timeColumns.includes(value) ) { timeColumnIndexes.push(i) };
+  });
+
+  // Add the nowrap class to cells with time column indexes
+  table.find('td').each(function() {
+    if (timeColumnIndexes.includes( $(this)[0].cellIndex )) {
+      $(this).addClass('nowrap')
```
Comment (jstirnaman) [inline assets/js/content-interactions.js:178]:
```suggestion
      $(this).addClass('nowrap');
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
+1 for JS and CSS instead of the shortcode.

File: assets/js/content-interactions.js (hunk 2 above)
Comment (sanderson) [reply assets/js/content-interactions.js:160]: Me and my semi-colons 🤦🏻

File: assets/js/content-interactions.js (hunk 1 above)
Comment (sanderson) [reply assets/js/content-interactions.js:153]: Yeah, we could revisit all the js and strip out jquery. We can save it for another line of work.

**Pull request merged**